fix(curriculum): remove media query requirement, general improvements (#46639)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/61539e07e7430b528fbffe21.md

# --description--

> Give your `.gallery` selector a `padding` property set to `20px 10px`.

Reviewer: we need the reason, of why they are adding `padding` so they don't get lost while searching for it in the frame.

Author: I don't understand this comment?

Reviewer: We are asking them to do something, and we aren't telling them what that something for, so they are left to guess why they did something, instead of how to implement something in the future

Author: I updated it but I will say I did it more to avoid the code block near punctuation. I think there must be plenty of steps in many challenges that are not explained in that way.

Reviewer: I will create an issue for them, after this pr. 👍

> Give your `.gallery` selector a `padding` property set to `20px 10px` as the value.
> 
> Then give the `.gallery` element a `max-width` of `1400px` and center it using an auto `margin` by setting `0 auto` as the `margin` value.

Reviewer 2: ```suggestion
Give your `.gallery` selector a `padding` property set to `20px 10px`.

Then, give it a `max-width` of `1400px` and center by setting its `margin` to `0 auto`.
```

Author: End of sentence code block.

Reviewer 2: updated

body {
  margin: 0;
  font-family: sans-serif;
  background: #f5f6f7;
}

.header {
  text-align: center;
  text-transform: uppercase;
  padding: 32px;
  background-color: #0a0a23;
  color: #fff;
  border-bottom: 4px solid #fdb347;
}

--fcc-editable-region--
.gallery {
  display: flex;

>   flex-direction: row;

Reviewer: ```suggestion
  flex-direction: row-reverse;
```
